### ISSUE TITLE
FuelLines connected to moveable parts load in the built position after f5/f9

### ISSUE THREAD

**Issue by numberlesstim** [KSPModdingLibs/KSPCommunityFixes#201]
If you attach a FuelLine to a movable part (something connected to a rotorServo for example), it will not remember the correct position after f5/f9. Instead it will use the positions it did have in the editor. This happens both with and without KSPCF.

Repro steps:

1. Load the provided craft, or build it yourselve (see pics or instructions below)
2. Spawn it and let the bar rotate a bit
3. Apply the brakes to stop it in a position different from the editor
4. (optional) Lock the servo
5. F5/F9

Note: It does not matter if the FuelLine collides with a part or not before f5/f9.

Craft:
[FuelLineBug.zip](https://github.com/KSPModdingLibs/KSPCommunityFixes/files/14360164/FuelLineBug.zip)
1. Place cockpit and attach a servoRotor
2. Add a structural beam to the rotor
3. Connect a FuelLine from the cockpit to the beam

![craft](https://github.com/KSPModdingLibs/KSPCommunityFixes/assets/117858473/ba760a64-266b-47be-905d-f197a2f33463)

![f5f9](https://github.com/KSPModdingLibs/KSPCommunityFixes/assets/117858473/c9e089ad-3870-4cd2-ae78-865c0e78caad)

**Comment by JonnyOThan** [comment]
Did you try a repro without tweakscale or other mods?  There's a high chance of interference there.

**Comment by numberlesstim** [comment]
Yes, i did test without any mods. Same thing happens. I only already had the images taken...

**Comment by numberlesstim** [comment]
Thats sad news. Guess i'll have to shelve my rowboat then.
![Rowboat](https://github.com/KSPModdingLibs/KSPCommunityFixes/assets/117858473/219d0a09-5933-4853-8d08-625ba7e7d99b)
